src/bundler.ts

```diff
@@ -65,21 +54,16 @@ const bundle = async (
       functions,
       importMap,
     }),
+    bundleESZIP({
```

Comment:
This means that we'll always generate two bundles (JS and ESZIP), which in the long term we don't want to do. What do you think about changing the logic so that:

- we use `true` as the default value for `edge_functions_produce_eszip`, so that it'll be enabled for CLI deployments
- if the `edge_functions_produce_eszip` flag is enabled, we generate _only_ ESZIP
- if the `edge_functions_produce_eszip` flag is disabled, we generate _only_ JS — this allows us to still flip a switch and go back to JS bundles in buildbot if something goes wrong

Reply:
Ohhhh gotcha, yeah makes sense to me, good shout!

Reply:
Fixed: https://github.com/netlify/edge-bundler/pull/50/commits/da27e7af839dc85c4e88ba0d4b65d50728575ecc